model/src/main/kotlin/config/ProjectExclude.kt

```diff
@@ -33,12 +33,8 @@ data class ProjectExclude(
         @JsonInclude(JsonInclude.Include.NON_NULL)
         val comment: String?,
         @JsonInclude(JsonInclude.Include.NON_EMPTY)
-        val packages: List<PackageExclude> = emptyList(),
-        @JsonInclude(JsonInclude.Include.NON_EMPTY)
-        val scopes: List<ScopeExclude> = emptyList(),
-        @JsonInclude(JsonInclude.Include.NON_EMPTY)
-        val errors: List<ErrorExclude> = emptyList()
+        val scopes: List<ScopeExclude> = emptyList()
```

Comment:
Do make the below implementation of `exclude` more understandable, I believe we must add some comments here and / or below. Something like:

"The scopes the exclude is limited to. If empty, the whole project is excluded, otherwise only the contained scopes in this project are excluded."

I just saw that something similar is already in the class docs, but IMO we should move these docs to the individual properties. Can be done in a separate PR, though.

Reply:
Ok, just saw that you're documenting basically just that in the next commit.